src/net.cpp

```diff
@@ -2377,6 +2406,17 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
         }
     }
 
+    m_anchors = ReadAnchors(GetDataDir() / ANCHORS_DATABASE_FILENAME);
+    Assert(MAX_BLOCK_RELAY_ONLY_ANCHORS >= 0);
+    constexpr size_t max_anchor_num = static_cast<size_t>(MAX_BLOCK_RELAY_ONLY_ANCHORS);
```

Comment:
Why not just make the constexpr `MAX_BLOCK_RELAY_ONLY_ANCHORS` a `size_t` rather than use this cast?

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693330527).